opencensus/stats/exporters/stackdriver_exporter.py

```diff
@@ -432,10 +431,10 @@ def get_task_value():
     """ getTaskValue returns a task label value in the format of
      "py-<pid>@<hostname>".
     """
-    task_value = "py@" + str(os.getpid())
     hostname = platform.uname()[1]
-    task_value += hostname if hostname is not None else "localhost"
-    return task_value
+    if not hostname:
```

Comment:
I know that this also bloats the PR, but the existing implementation didn't conform to the comment. Happy to revert or send as a separate change.